app/src/androidTest/java/com/duckduckgo/app/cta/ui/CtaViewModelTest.kt

```diff
@@ -472,13 +474,13 @@ class CtaViewModelTest {
     }
 
     @Test
-    fun whenRefreshCtaOnHomeTabAndIntroCtaWasShownThenEndCtaShown() = coroutineRule.runBlocking {
+    fun whenRefreshCtaOnHomeTabAndIntroCtaWasShownThenFireproofCtaShown() = coroutineRule.runBlocking {
```

Comment:
We are missing the End Cta test now :) I think we need two more tests:

1. `whenRefreshCtaOnHomeTabAndIntroCtaWasShownThenEndCtaShown` as it was but only if the variant is not the experiment
2. `whenRefreshCtaOnHomeTabAndFireproofCtaWasShownThenEndCtaShown` if the variant is the experiment

Reply:
These tests have been added 👍